src/sync/sync_session.cpp

```diff
@@ -474,30 +500,18 @@ void SyncSession::handle_error(SyncError error)
             }
             case ProtocolError::illegal_realm_path:
             case ProtocolError::no_such_realm:
+                break;
             case ProtocolError::permission_denied:
+                update_error_and_mark_file_for_deletion(error, ShouldBackup::no);
```

Comment:
This really, really, really needs to be checked to ensure it's not a permission or management Realm.